crates/aptos-logger/src/aptos_logger.rs

```diff
@@ -724,6 +724,28 @@ impl Writer for FileWriter {
     }
 }
 
+pub struct MockWriter {
```

Comment:
these could also be under `#[cfg(test)]` hiding?

Reply:
Or inside the tests mod?

Reply:
Moved this into the tests module